.travis.yml

```diff
@@ -16,8 +16,8 @@ after_success:
   - true
 
 before_deploy:
-  - if [ -n "$TRAVIS_TAG" -a "$TRAVIS_BRANCH" = "master" ]; git archive HEAD -o $TRAVIS_TAG.tar.gz; fi
-  - if [ -n "$TRAVIS_TAG" -a "$TRAVIS_BRANCH" = "master" ]; git archive HEAD -o $TRAVIS_TAG.zip; fi
+  - if [ -n "$TRAVIS_TAG" ]; git archive HEAD -o $TRAVIS_TAG.tar.gz; fi
```

Comment:
I think this needs a `then` after `]` ... I just get error locally for...

```if [ -n ""]; echo hello; fi```
.... but it works with 
```if [ -n "" ]; then echo hello; fi```

Reply:
huh; has worked in previous travis runs. will push an update later today though.